src/main.ts

```diff
@@ -35,6 +40,20 @@ function run(): void {
       )}`
     );
 
+    core.debug('Switching to the correct Angular Builder');
+    const angularJson: AngularJSON = JSON.parse(
+      fs.readFileSync(angularJsonPath).toString()
+    );
+    const modifiedAngularJson = switchAngularBuilder(
+      angularVersion,
+      angularJson
+    );
+    fs.writeFileSync(
+      angularJsonPath,
+      JSON.stringify(modifiedAngularJson, null, 2)
+    );
+    core.debug('Correct Angular Builder selected');
```

Comment:
suggestion: extract this into a separate function. The `run` function is already doing way too much.